code/modules/reagents/chemistry_reagents/medical.dm

```diff
@@ -1,986 +1,1066 @@
+			if(E && istype(E))
+				if(E.damage > 0)
+					E.damage = max(E.damage - 1, 0)
+
+	on_overdose(mob/living/M)
+		M.apply_damage(2, TOX)
+
+	on_overdose_critical(mob/living/M)
+		M.apply_damages(1, 1, 3)
+
+/datum/reagent/peridaxon
+	name = "Peridaxon"
+	id = "peridaxon"
+	description = "Used to stabilize internal organs while waiting for surgery. Medicate cautiously. Repairs organ damage at cryogenic temperatures."
+	reagent_state = LIQUID
+	color = "#C845DC"
+	overdose = REAGENTS_OVERDOSE/2
+	overdose_critical = REAGENTS_OVERDOSE_CRITICAL/2
+	custom_metabolism = 0.05
+	scannable = TRUE
+
+	/datum/reagent/peridaxon/on_mob_life(mob/living/M)
+		. = ..()
+		if(!.)
+			return
+		if(M.bodytemperature < 170)
+			if(ishuman(M))
+				var/mob/living/carbon/human/H = M
+				for(var/datum/internal_organ/I in H.internal_organs)
+					if(I)
+						if(I.robotic != ORGAN_ROBOT)
+							if(I.damage > 0)
+								I.damage = max(I.damage - 2*REM, 0)
+
+	on_overdose(mob/living/M)
+		M.apply_damage(2, BRUTE)
+
+	on_overdose_critical(mob/living/M)
+		M.apply_damages(3, 3, 3)
+
+/datum/reagent/bicaridine
+	name = "Bicaridine"
+	id = "bicaridine"
+	description = "Bicaridine is an analgesic medication and can be used to treat blunt trauma."
+	reagent_state = LIQUID
+	color = "#E8756C"
+	overdose = REAGENTS_OVERDOSE
+	overdose_critical = REAGENTS_OVERDOSE_CRITICAL
+	scannable = TRUE
+
+	on_mob_life(mob/living/M, alien)
+		. = ..()
+		if(!.) return
+		if(M.stat == DEAD)
+			return
+		M.heal_limb_damage(2*REM,0)
+
+	on_overdose(mob/living/M)
+		M.apply_damage(1, BURN)
+
+	on_overdose_critical(mob/living/M)
+		M.apply_damages(0, 4, 2)
+
+/datum/reagent/quickclot
+	name = "Quick Clot"
+	id = "quickclot"
+	description = "A chemical designed to quickly stop all sorts of bleeding by encouraging coagulation. Heals internal bleeding at cryogenic temperatures."
+	reagent_state = LIQUID
+	color = "#CC00FF"
+	overdose = REAGENTS_OVERDOSE/2 //Was 4, now 6 //Now 15
+	overdose_critical = REAGENTS_OVERDOSE_CRITICAL/2
+	custom_metabolism = 0.05
+	scannable = TRUE
+
+	/datum/reagent/quickclot/on_mob_life(mob/living/M)
+		. = ..()
+		if(!.)
+			return
+		if(M.bodytemperature < 170)
+			if(ishuman(M))
+				var/mob/living/carbon/human/H = M
+				for(var/datum/limb/L in H.limbs)
+					if(L)
+						for(var/datum/wound/W in L.wounds)
+							if(W.internal)
+								W.damage = max(0, W.damage - 2*REM)
+
+	on_overdose(mob/living/M)
+		M.apply_damage(3, BRUTE)
+
+	on_overdose_critical(mob/living/M)
+		M.apply_damages(2, 3, 3)
+
+/datum/reagent/hyperzine
+	name = "Hyperzine"
+	id = "hyperzine"
+	description = "Hyperzine is a highly effective, long lasting, muscle stimulant.  May cause heart damage"
+	reagent_state = LIQUID
+	color = "#C8A5DC" // rgb: 200, 165, 220
+	custom_metabolism = 0.2
+	overdose = REAGENTS_OVERDOSE/5
+	overdose_critical = REAGENTS_OVERDOSE_CRITICAL/5
+	scannable = TRUE
+
+	on_mob_life(mob/living/M)
+		. = ..()
+		if(!.) return
+
+		M.reagent_move_delay_modifier -= 9

```

Comment:
Ultrazine is 10. Lower this to 5 or so.

Reply:
Ultrazine also lasts forever, has a higher OD ceiling, and has downsides that don't come online until after relevance.

Reply:
Besides being more rare and a bargaining tool of the Liason, yea.